### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by GengDavid** [Zch0414/hlip#5]
Hi, @Zch0414, thanks for sharing your excellent work. 

I am wondering that is the released pre-trained CT-RATE model trained with qwen_organ_annotation.json and siglip loss?

**Comment by GengDavid** [comment]
Great to know that. Thanks for your promptly reply!

**Comment by Zch0414** [comment]
Hi,

The example script uses a batch size of 64 in total with 4 GPUs. You can set `--batch-size 32 --accum-batch 4 --lr 8e-5 --warm-up 47` to try a batch size of 512.

**Comment by GengDavid** [comment]
Hi @Zch0414 
I’ve tried training with the following settings:
```--batch-size 32 --accum-batch 4 --lr 8e-5 --warm-up 47```
However, I'm only able to achieve 75.8 AUROC.

Do you have any further suggestions on how I might improve the performance and align it more closely with the reported results?

**Comment by Zch0414** [comment]
Hi David,

I noticed that your loss decreases more slowly than mine—I was able to achieve a final loss below 4. I can rerun the repository late next week and share my results with you then. Does that timeline work for you?

**Comment by GengDavid** [comment]
Thanks for letting me know. After double-checking my code, I found I'm using a version before the update in #4. Could that be the reason for the issue?

Also, great, looking forward to seeing your results!

**Comment by Zch0414** [comment]
Hi David,

Regarding issue #4, it will not affect performance since it does not alter the actual forward logic for single-scan inputs.
I’ve rerun the repository and obtained results similar to those reported in the paper:

<img width="4550" height="2390" alt="Image" src="https://github.com/user-attachments/assets/8c2ee4cf-9925-46fb-80ac-1c2d6b790a6f" />
<img width="4550" height="2390" alt="Image" src="https://github.com/user-attachments/assets/33bc7182-3114-4305-b6a4-10283882bea9" />
